### PR TITLE
Update `docs/polaris/data-science-workflows/python.md`

### PR DESCRIPTION
**Pull request by saforem2**
Update Python documentation on Polaris

- Content changes:
  - Removed text about specific versions etc to try and make as generic (and future-proof 🤞) as possible
  - Use `module load conda ; conda activate base` instead of split across lines (ensure `base` **activated**)
  - Remove instructions about creating `venv` **without** `--system-site-packages` (was confusing IMO)
- Formatting changes:
  - Wrap lines at 80 characters

### CODE REVIEW

**Comment by felker** [comment]
What is the motivation for wrapping lines at 80 characters? That seems inconsistent with the rest of the repo, and am not sure adding hard line wraps is a best practice for version controlling Markdown...

**Comment by saforem2** [comment]
> What is the motivation for wrapping lines at 80 characters? That seems inconsistent with the rest of the repo, and am not sure adding hard line wraps is a best practice for version controlling Markdown...


Yeah so really my main thought was to try and enforce at most one sentence per line, similar to writing LaTeX, e.g.

1. [`markdownlint`](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md):

    > Rationale: Extremely long lines can be difficult to work with in some editors. 
    > More information: https://cirosantilli.com/markdown-style-guide#line-wrapping.
   

2. [Semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line):

    > By starting a new line at the end of each sentence, and splitting sentences themselves at natural breaks between clauses,
    > a text file becomes far easier to edit and version control.[^1]

3. [HN: Writing one sentence per line](https://news.ycombinator.com/item?id=31808093)
4. [r/LaTeX: Using LaTeX with Version Control](https://www.reddit.com/r/LaTeX/comments/2mra3o/using_latex_with_version_control/)

but it really is just a personal / stylistic preference at the end of the day 🤷🏻‍♂️

**Comment by saforem2** [comment]
@felker let me know if there is anything else about the content that isn't clear or should be updated

we can table the formatting discussion for now 😂 but otherwise I think the rest of it should be OK

**Review by felker — APPROVED** [review]

File: docs/polaris/data-science-workflows/python.md [hunk 1]
```diff
@@ -1,88 +1,111 @@
...
-(base) $ which python3
-/soft/datascience/conda/2022-09-08/mconda3/bin/python3
+module load conda ; conda activate base
 ```
-In one line, `module load conda; conda activate`. This can be performed on a compute node, as well as a login node. 
 
-As of writing, the latest `conda` module on Polaris is built on Miniconda3 version 4.14.0 and contains Python 3.8.13. Future modules may contain entirely different major versions of Python, PyTorch, TensorFlow, etc.; however, the existing modules will be maintained as-is as long as feasible. 
+This will load and activate the base environment.
 
-While the shared Anaconda environment encapsulated in the module contains many of the most commonly used Python libraries for our users, you may still encounter a scenario in which you need to extend the functionality of the environment (i.e. install additional packages)
```
Comment (felker) [inline docs/polaris/data-science-workflows/python.md:20]: just curious, why delete this?

Comment (saforem2) [reply docs/polaris/data-science-workflows/python.md:20]: honestly I think it maybe just sounded a bit wordy ?? idk
